Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM golang:1.4
+
+ENV GOPATH /go
+ENV PATH ${PATH}:/go/bin
+ENV TARGET /source
+ENV GOPATH_TARGET /go/src/github.com/docker/docker-network
+
+ENV SUBSYSTEMS namespace
+
+RUN go get -u github.com/kr/godep
+RUN mkdir ${TARGET}
+RUN mkdir -p $(dirname ${GOPATH_TARGET}) && \ 
```

Comment:
You don't, you call `make build` and build it yourself. `make test`
calls `make build` for you each run.

On Mon, 2015-01-05 at 18:31 -0800, Alexander Morozov wrote:
> In Dockerfile:
> 
> > @@ -0,0 +1,17 @@
> > +FROM golang:1.4
> > +
> > +ENV GOPATH /go
> > +ENV PATH ${PATH}:/go/bin
> > +ENV TARGET /source
> > +ENV GOPATH_TARGET /go/src/github.com/docker/docker-network
> > +
> > +ENV SUBSYSTEMS namespace
> > +
> > +RUN go get -u github.com/kr/godep
> > +RUN mkdir ${TARGET}
> > +RUN mkdir -p $(dirname ${GOPATH_TARGET}) && \ 
> 
> And how is supposed to distribute this image through hub? I am
> personally -1 on this. There is just no reason to force people use
> Makefile instead of just docker. Or docker instead of just go test.
> 
> —
> Reply to this email directly or view it on GitHub.
> 
>